Sources/Build/BuildManifest/LLBuildManifestBuilder+Swift.swift

```diff
@@ -376,6 +376,9 @@ extension LLBuildManifestBuilder {
         let cmdName = target.getCommandName()
 
         self.manifest.addWriteSourcesFileListCommand(sources: target.sources, sourcesFileListPath: target.sourcesFileListPath)
+        let outputFileMapPath = target.tempsPath.appending("output-file-map.json")
+        // FIXME: Eliminate side effect.
+        try target.writeOutputFileMap(to: outputFileMapPath)
```

Comment:
Why not restructure this to avoid the side effect in the first place? There are plenty of tests that rely on this being side-effect-free, and now these tests will unintentionally write these output files unexpectedly.